src/orca/errors.py

```diff
@@ -9,8 +9,8 @@ class ClientRequestError(OrcaError):
     """Client request failed."""
 
 
-class ClientArgsError(OrcaError):
-    """Client arguments are missing or invalid."""
+class ClientAttrError(OrcaError):
```

Comment:
I forgot about `AttributeError`. I agree that this would be a better exception to use (or inherit from for a more specific custom exception class). That said, I update this class in #11 to `ConfigError` ([source](https://github.com/Sage-Bionetworks-Workflows/py-orca/pull/11/files#diff-a96ad2c0eb692c5a8d2370cb1dfaca8d2afea0f7b72a4873dd054dcf632fce38L12-R12)). I'll update that exception class to inherit from `AttributeError`.